cypress/integration/ambianic-tests/settings.spec.js

```diff
@@ -30,4 +30,19 @@ context('Settings', () => {
     it('Should have a row', () => {
       cy.get('[data-cy=template-row]').should('exist')
     })
+
+    it("It display connected edge version after connection", () => {
+      const version = '1.0.0'
+
+      cy.get('[data-cy=sendRemotePeerID]').click()
+
+      cy.window().then(win => {
+        win.__store__.commit('PEER_CONNECTED', null)
+        win.__store__.commit('EDGE_DEVICE_DETAILS', { 
+          version
+         })
+
+        cy.get('[data-cy=title-text]').should('contain.text', version )
```

Comment:
@vickywane did you check whether this test fails if the assertion fails. I am not sure whether cypress will fail when the assertion is embedded in a promise or it will just always pass and ignore the result of the assertion. It seems to have different evaluation semantics than jest.
https://docs.cypress.io/api/commands/window#No-Args

Reply:
Yes. 

I just tested this particular e2e test. It fails when the version text is not found.

Reply:
OK. Thank you for verifying.